kotlin-inject-compiler-core/src/main/kotlin/me/tatarka/inject/compiler/Ast.kt

```diff
@@ -58,6 +58,8 @@ abstract class AstClass : AstElement(), AstHasModifiers {
 
     abstract val primaryConstructor: AstConstructor?
 
+    abstract val constructors: List<AstConstructor>?
```

Comment:
Is there a reason this is nullable? I'd expect no constructors to be represented as an empty list.

Reply:
No reason. For some reason, I had an idea that it was the library preference (vs empty collections) but I now I can't find anything confirming that. Will switch over. Thank you.